Chinese/dream/dream.md

```diff
@@ -0,0 +1,7 @@
+你來到這就是在作夢~ 來吧! 以下自由選擇:
+
+1. [我要穿越](https://github.com/xbane123/create-your-own-adventure/blob/master/Chinese/Yifan/modernCity.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.